docs/en/sql-reference/statements/alter/user.md

```diff
@@ -15,11 +15,23 @@ ALTER USER [IF EXISTS] name1 [ON CLUSTER cluster_name1] [RENAME TO new_name1]
     [NOT IDENTIFIED | IDENTIFIED {[WITH {no_password | plaintext_password | sha256_password | sha256_hash | double_sha1_password | double_sha1_hash}] BY {'password' | 'hash'}} | {WITH ldap SERVER 'server_name'} | {WITH kerberos [REALM 'realm']}]
     [[ADD | DROP] HOST {LOCAL | NAME 'name' | REGEXP 'name_regexp' | IP 'address' | LIKE 'pattern'} [,...] | ANY | NONE]
     [DEFAULT ROLE role [,...] | ALL | ALL EXCEPT role [,...] ]
+	[GRANTEES {user | role | ANY | NONE} [,...] [EXCEPT {user | role} [,...]]]
     [SETTINGS variable [= value] [MIN [=] min_value] [MAX [=] max_value] [READONLY | WRITABLE] | PROFILE 'profile_name'] [,...]
 ```
 
 To use `ALTER USER` you must have the [ALTER USER](../../../sql-reference/statements/grant.md#grant-access-management) privilege.
 
+## GRANTEES Clause {#grantees}
+
+Specifies users or roles which are allowed to receive [privileges](../../../sql-reference/statements/grant.md#grant-privileges) from this user on the condition this user has also all required access granted with [GRANT](../../../sql-reference/statements/grant.md) option. Options of the `GRANTEES` clause:
+
+-   `user` — Specifies user which is allowed to receive privileges from this user.
+-   `role` — Specifies role which is allowed to receive privileges from this user.
+-   `ANY` — User with `GRANT` option can provide privileges to anyone. Used by default.
```

Comment:
All these options requires this user to have all the required privileges with GRANT option, and you already said that a few lines earlier. So I think this way is better:

```
user - Specifies a user this user can grant privileges to.
role - Specifies a role this user can grant privileges to.
ANY - This user can grant privileges to anyone. It's the default setting.
NONE - This user can grant privileges to none.
```